.github/workflows/continuous-integration.yml

```diff
@@ -31,6 +31,8 @@ jobs:
             dbal-version: "2.13"
           - php-version: "8.1"
             dbal-version: "3@dev"
+          - php-version: "8.2"
+            dbal-version: "3@dev"
```

Comment:
As long as doctrine/dbal#5381 is not released, the tests will fail on PHP 8.2 if we don't test against a dev snapshot.